python/taichi/_ti_module/__init__.py

```diff
@@ -1,13 +1,14 @@
 import argparse
 import json
+import runpy
 from pathlib import Path
 from typing import List
 
+from taichi._lib import core as _ti_core
 from taichi._ti_module.cppgen import generate_header
+from taichi.aot._export import _aot_kernels
 from taichi.aot.conventions.gfxruntime140 import GfxRuntime140
 
-import taichi as ti
-
 
 def module_cppgen(parser: argparse.ArgumentParser):
```

Comment:
nit: move implementations out of `__init__.py` to avoid unexpected leak during import chain? ;)